torch/utils/hooks.py

```diff
@@ -12,18 +12,29 @@ class RemovableHandle(object):
     id: int
     next_id: int = 0
 
-    def __init__(self, hooks_dict: Any) -> None:
+    def __init__(
+        self, hooks_dict: Any, *, kwargs_dict: Dict[int, bool] = None
```

Comment:
Not sure what `kwargs_dict` means in this context. `extra_dict` sounds better.
Also the typing here should be the same as `hooks_dict` no?

Even better might be to just do `*args` and take as many dictionary as the user wants.

Reply:
Let me do `extra_dict` for now. Can add arbitrary numbers of dict if necessary in followups.